common/types.ts

```diff
@@ -65,25 +59,72 @@ export interface Scene {
   // TODO
 }
 
-// ---- Light State ----
+// ---- Pattern Types ----
 
-export interface RoomLightState {
-  power: boolean;
-  scene: Scene;
+export enum PatternType {
```

Comment:
I'm not sure why these old RoomLightState duplicates have been hanging around in git diffs...I already removed them at https://github.com/rvl-system/home-lights/commit/76279e11d0cab4e65670b65899b2f41fe9913b8b#diff-0f180c10ed511051613ddbf201bcba5544a0193176b29b331140dce71d039c12. Strange.